src/main/java/com/nextstep/racingcargame/core/Car.java

```diff
@@ -40,7 +38,13 @@ public boolean isLongerThan(Distance distance) {
     }
 
     public void move(MovingStrategy movingStrategy) {
-        this.distance.moveForward(movingStrategy.randomNumber());
+        if (isGoForwardNumber(movingStrategy.randomNumber())) {
+            this.distance = this.distance.moveForward();
+        }
```

Comment:
자동차를 움직이게 하는 조건을 MovingStrategy에서 관리하는 방법도 고민해 보시면 좋을 것 같아요
참고만 해주세요 😃 
```suggestion
        if (movingStrategy.movable())) {
            this.distance = this.distance.moveForward();
        }
```